nfv_sriov.md

```diff
@@ -0,0 +1,253 @@
+# NFV SRIOV
+
+This document describes how to deploy EDPM nodes with SRIOV.
+
+## Procedure Overview
+
+1. Configure kernel args and tuned parameters
+2. Configure SRIOV parameters
+3. Configure the networks with SRIOV interfaces of the EDPM nodes
+2. Configure Nova and Neutron parameters.
+
+In order to complete the above procedure, the `services` list of the
+`OpenStackDataPlaneNodeSet` CR needs to be edited.
+
+## OpenStackDataPlaneNodeSet services list
+
+EDPM nodes can be configured by creating an
+`OpenStackDataPlaneNodeSet` CR which the
+[dataplane-operator](https://openstack-k8s-operators.github.io/dataplane-operator)
+will reconcile to create OpenStackDataPlaneService resources
+when an `OpenStackDataPlaneDeployment` CR is created.
+These types of CRs have a `services` list like the following:
```

Comment:
It is apparent from the example. But simply `OpenStackDataPlaneNodeSet` would work here.

Reply:
done